bugtool/cmd/configuration.go

```diff
@@ -201,6 +198,19 @@ func catCommands() []string {
 	return commands
 }
 
+// routeCommands gets the routes dynamically.
+func routeCommands() []string {
+	commands := []string{}
+	routes, _ := execCommand("ip route show table all | grep 'table' | sed 's/.*\\(table.*\\)/\\1/g' | awk '{print $2}' | uniq | grep -e '[0-9]'")
```

Comment:
One thing you could do here to reduce the amount of times we pipe to another command is to use `grep --only-matching`. This can replace the `grep` pipe to `sed. 

`grep --only-matching "table [0-9]"` likely gets us what we want. Worth trying out.

Reply:
@christarazi cheers, indeed much cleaner ;) 
Btw I used the `-E` flag for grep to use the repetition in regex as we can have a table ID of up to 9 digits. Not sure about the portability but it's cleaner that [0-9] repeated a couple of times.

Reply:
Re: portability, I'm not sure about that either. I would imagine that `grep -E` isn't different across platforms. However, this tool executes where Cilium is running, which only supports Linux currently, so we don't need to worry about that.

Is there a reason to choose `{}` over `+`, i.e. `table [0-9]+`?

Reply:
Perfect.
About the `{}`, I would say in this scenario not really, so I've switched it to `+` as it is simpler.